### DISCUSSION TITLE
giscus-replaces-disqus

### DISCUSSION THREAD

**Discussion by giscus** [yshmarov/yshmarov.github.io#8] [General]
# giscus-replaces-disqus

I’ve been using Disqus comments on my blog for years.

https://blog.corsego.com/giscus-replaces-disqus

**Comment by secretpray** [comment]
Чудово!
Будемо тепер набридати у цьому чаті!)

**Comment by alec-c4** [comment]
Hey! You know, giscus requires quite unusual access permissions. I suspect it’s a bug, but it demands access to discussions on every repository on your account, not just the ones you use for your blog.

**Reply by yshmarov** [reply]
only selected repositories

<img width="669" alt="Screenshot 2025-01-27 at 13 12 31" src="https://github.com/user-attachments/assets/9e57fcb2-8fe3-4f91-9132-0cfddaa4d648" />

**Reply by alec-c4** [reply]
It’s quite peculiar because I spent several days trying to identify the issue with my repository. Initially, I encountered an error stating that not all requirements were met. However, after granting access to all repositories, the issue resolved. Interestingly, after changing access back to only one repository, the problem was gone.
